internal/core/graphics/brush.rs

```diff
@@ -112,12 +114,13 @@ impl Brush {
                     GradientStop { color: s.color.brighter(factor), position: s.position }
                 })))
             }
-            Brush::ConicGradient(g) => {
-                Brush::ConicGradient(ConicGradientBrush::new(g.stops().map(|s| GradientStop {
+            Brush::ConicGradient(g) => Brush::ConicGradient(ConicGradientBrush::new(
```

Comment:
Since this touches only the colors, and not the stops, this could just map the colors without having to re-normalize thing.
This doesn't have to be done in this PR, but we could imagine a (internal) `Brush::visit_colors(&mut self, map: impl Fn(&mut Color))` and so one could implement `brighter`, `darker`, `with_alpha` and `transparentize` using it.  But anyway, this can be done independently.

Reply:
Yes. they are redundant. I'll improve it later.